tests/manage/monitoring/prometheus/test_deployment_status.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import time
+
+import ocs_ci.utility.prometheus
+
+
+log = logging.getLogger(__name__)
+
+
+def test_ceph_manager_stopped(workload_stop_ceph_mgr):
+    """
+    Test that there is appropriate alert when ceph manager
+    is unavailable and that this alert is cleared when the manager
+    is back online.
+    """
+    prometheus = ocs_ci.utility.prometheus.PrometheusAPI()
+
+    alerts_response = prometheus.get(
+        'alerts',
+        payload={
+            'silenced': False,
+            'inhibited': False,
+            'start': int(workload_stop_ceph_mgr['start']),
+            'stop': int(workload_stop_ceph_mgr['stop']),
+        }
+    )
+    assert alerts_response.ok is True
```

Comment:
please add assertion message in case of failure

Reply:
done